packages/ui/src/TextArea.module.scss

```
@@ -0,0 +1,89 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.container {
+  @include c.typographyBodySmall;
+
+  color: c.$colorText;
+  box-sizing: border-box;
+  margin: 0;
+
+  .label {
+    margin-bottom: c.$grid;
+  }
+
+  .helperText {
+    display: block;
+    margin: c.$grid * 2 0 0 c.$grid * 2;
```

Help also has internal padding of `c.$grid`. So with this margin we have `3*c.$grid` left spacing in total. For the TextField  it is only `2*c.$grid`. Do you think it would make sense to match the TextField spacing here as well?
```suggestion
    margin: c.$grid 0 0 c.$grid;
```

---

Good catch!